Amplify/Categories/DataStore/Model/Collection/List+Model.swift

```diff
@@ -85,35 +92,35 @@ open class List<ModelType: Model>: ModelList {
             }
         }
 
-        let json = try JSONValue(from: decoder)
-
-        switch json {
-        case .array:
-            let elements = try Elements(from: decoder)
-            self.init(elements)
-        default:
-            self.init(Elements())
-        }
+        self.init()
     }
 
-    public func encode(to encoder: Encoder) throws {
+    open func encode(to encoder: Encoder) throws {
+        let elements = [ModelType]()
         try elements.encode(to: encoder)
     }
 
     // MARK: - Asynchronous API
 
-    /// Use to initialize the collection.
+    /// Call this to initialize the collection if you have retrieved the list by traversing from your model objects
+    /// to its associated children objects. For example, a Post model may contain a list of Comments. By retrieving the
+    /// post object and trvasering to the comments, the comments are not retrieved from the data source until this
+    /// method is called. Data will be retrieved based on the plugin's data source and may have different failure
+    /// conditions, such as one that requires connectivity may fail with a network error. You can alternatively access
+    /// the Collection methods on the list to trigger the retrieval of data, but also expect that any operation to
+    /// perform as fast as it takes to make a request to the datastore and back.
     ///
-    /// - seealso: `load()`
+    /// If you have directly created this list object then the collection has already been initialized and calling this
```

Comment:
Suggested reword. Please fill in with the appropriate initializer spelling and reformat comment as appropriate

```suggestion
    /// If you have directly created this list object (for example, by calling `List(elements:)` then the collection has already been initialized and calling this
```